<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RenameComplex</class>
 <widget class="QWidget" name="RenameComplex">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>214</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SeedComplex</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>150</y>
     <width>156</width>
     <height>24</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelOldName">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>80</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Old name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="newName">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelNewName">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>80</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>New name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="renameWindowTitle">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Renaming simplicial complex</string>
   </property>
  </widget>
  <widget class="QLabel" name="oldName">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>251</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
